source/contributing/how-to-migrate.md

```diff
@@ -0,0 +1,34 @@
+# How to Migrate Documentation - Step-by-Step Process
+
+Migrating documentation is often crucial when reorganizing any project. As such, below is a list of instructions and guidelines to aid you when embarking on the migration journey.
+
+1. **Licensing**
+   1. Familiarize yourself with the licenses governing the documentation you intend to migrate.
+   2. Verify if the license of the documentation is compatible with this project's current license.
+   3. If the licenses align, proceed with the migration. Otherwise, follow the steps below.
+   4. Identify the file and determine all contributors to the documentation (typically using blame or a co-owners document).
+   5. Contact all contributors, requesting permission to migrate the document to the new license.
+   6. Await responses from all recipients and obtain explicit approval from each contributor before proceeding.
+   7. If agreement from all contributors cannot be obtained, consider alternative solutions to avoid licensing conflicts, such as:
+        - A full rewrite of the document.
+        - Rewriting the areas of specific contributors who did not reply or approve.
+
+2. **Documentation Assessment**

```

Comment:
You also need to consider which type of documentation category the material will end up being e.g. tutorial, recipe, concept, or reference. If the material doesn't clearly fall into one of these categories you may need to split it up, rewrite it, or abandon it entirely.